tests/FamilyHubs.ServiceDirectory.Api.AcceptanceTests/Tests/Stories/ServiceSearchTests.cs

```diff
@@ -6,28 +6,91 @@ namespace FamilyHubs.ServiceDirectory.Api.AcceptanceTests.Tests.Stories;
 
 // Define the story/feature being tested
 [Story(
-    AsA = "user of the metrics api",
-    IWant = "to be able to update usage information for postcode searches and filters",
-    SoThat = "I can record how search functionality is being used")]
+   AsA = "user of the metrics api",
+   IWant = "to be able to update usage information for postcode searches and filters",
+   SoThat = "I can record how search functionality is being used")]
 
 [TestClass]
 public class ServiceSearchTests
 {
-    private readonly ServiceSearchSteps steps;
+   private readonly ServiceSearchSteps steps;
 
-    public ServiceSearchTests()
-    {
-        //Get instances of the steps required for the test
-        steps = new ServiceSearchSteps();
-    }
+   public ServiceSearchTests()
+   {
+       //Get instances of the steps required for the test
+       steps = new ServiceSearchSteps();
+   }
 
-    //Add all tests that make up the story to this class.
-    [TestMethod]
-    public void Users_Can_Update_Metrics_For_Postcode_Search()
-    {
-        this.Given(s => steps.GivenIHaveAPostcodeToSearchWithinFind())
-            .When(s => steps.WhenISendARequest())
-            .Then(s => steps.Then200StatusCodeReturned())
-            .BDDfy();
-    }
+   //Add all tests that make up the story to this class.
+   [TestMethod]
+   public void Initial_Postcode_Search_as_a_Find_User()
+   {
+       this.Given(s => steps.GivenIPostToTheServiceSearchEndpointWithParameters("15","E1 2EN","200","1","2"))
+           .When(s => steps.WhenISendARequest())
+           .Then(s => steps.Then200StatusCodeReturned())
+           .BDDfy();
+   }
+  
+   [TestMethod]
+   public void Initial_Postcode_Search_as_a_Connect_User()
+   {
+       this.Given(s => steps.GivenIPostToTheServiceSearchEndpointWithParameters("15","E1 2EN","200","1","1"))
```

Comment:
refactor: The given isn't where we send the request we just build the request there so maybe it would be better to have something like 

GivenIHaveASearchServiceRequest(params)